src/entry.js

```diff
@@ -1,59 +1,50 @@
-// Promise polyfill to avoid "Promise is undefined" error in IE
-require('es6-promise').polyfill()
-
+/* eslint-disable import/first */
 // Object.assign() polyfill for IE (used in the reducer)
 import './helpers/es6-polyfill.js'
-import { fetchJSON } from './helpers/fetch'
+// Promise polyfill to avoid "Promise is undefined" error in IE
+require('es6-promise').polyfill()
 
 import React from 'react'
-import { render } from 'react-dom'
 import { AppContainer } from 'react-hot-loader'
-// import { whyDidYouUpdate } from 'why-did-you-update'
-
-// if (process.env.NODE_ENV !== 'production') {
-//   whyDidYouUpdate(React)
-// }
+import { render } from 'react-dom'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'react-dom'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->